src/com/company/Main.java

```diff
@@ -1,8 +1,134 @@
 package com.company;
 
+import java.util.*;
```

Comment:
Don't use wild cards in imports.

Reply:
Java put "*" cards automatically after 5 (approximately) imports from the same library (in my IDEA)

Reply:
Typed import manualy